test/common_plat/validation/api/init/init.c

```diff
@@ -24,10 +24,10 @@ static int odp_init_log(odp_log_level_t level, const char *fmt, ...);
 void init_test_odp_init_global_replace_abort(void)
 {
 	int status;
-	struct odp_init_t init_data;
+	odp_init_t init_data;
 	odp_instance_t instance;
 
-	memset(&init_data, 0, sizeof(init_data));
+	odp_init_param_init(&init_data);
```

Comment:
Since this is a void function there's no return code to check. We just want to be sure it's included in the code coverage report, hence these changes. It's basically doing the same memset() that's being replaced here, but in an implementation-independent manner. 

The other odp_xxx_param_init() routines follow the same practice in the validation tests, so there's nothing new here.